cmd/main.go

```diff
@@ -125,8 +124,13 @@ func request(meth method, pth string, pars params) (*http.Response, error) {
 	if pars.ctype != "" {
 		req.Header.Set("Content-Type", pars.ctype)
 	}
-	client := &http.Client{}
-	return client.Do(req)
+	tr := &http.Transport{}
+	client := &http.Client{Transport: tr}
+	res, err := client.Do(req)
+	cancel := func() {
+		tr.CancelRequest(req)
+	}
```

Comment:
Expose a cancel method to the convenience functions used by `sub` and others... this way we don't mem leak the server end when the client quits.